src/scaladoc/scala/tools/nsc/doc/html/resource/lib/template.css

```diff
@@ -593,7 +593,7 @@ div#definition > h4#signature > span.modifier_kind > i.unfold-arrow,
   background-color: #fff;
   margin: 5px 0;
   display: block;
-  font-family: "Source Code Pro";
+  font-family: "Source Code Pro", "Monaco", "Ubuntu Mono Regular", "Lucida Console", monospace;
```

Comment:
Yep, Lucida Console for Windows. Seemed to always be available. Always the first change I make in Git Bash.

![lucida-console-regular](https://cloud.githubusercontent.com/assets/1123855/13584825/2225410a-e4b0-11e5-818c-fe417d1efd36.png)